price-server/src/server/mod.rs

```diff
@@ -143,7 +143,12 @@ impl Price for PriceService {
             .app
             .get_cents_per_sats_exchange_mid_rate(Sats::from_major(req.amount_in_satoshis))
             .await?;
-        let ratio_in_cents_per_satoshis = f64::from_bits(mid_amount_in_cents);
+        let ratio_in_cents_per_satoshis = ConvertU64ToF64 {
```

Comment:
This struct for converting u64 -> f64 is not appropriate. Use stdlib or a cast.

Reply:
It should simply return the price of 1 sat as double. That implies a conversion from `impl From<UsdCent> for f64`.